circuit-benchmarks/src/evm_circuit.rs

```diff
@@ -83,7 +91,7 @@ mod evm_circ_benches {
             .parse()
             .expect("Cannot parse DEGREE env var as u32");
 
-        let circuit = TestCircuit::<Fr>::default();
+        let circuit = TestCircuit::<Fr>::new();
```

Comment:
I would like to suggest a different approach, that hopefully will be more robust in the future.  Building the witness of an empty block, like it is done in this test https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/dac2741dd36d45f3a3ded8fc782f657c936bb8be/zkevm-circuits/src/evm_circuit.rs#L416

So it should look like this:
```rust
let geth_data: GethData = TestContext::<0, 0>::new(None, |_| {}, |_, _| {}, |b, _| b)
    .unwrap()
    .into();
let mut builder =
    BlockData::new_from_geth_data_with_params(geth_data.clone(), CircuitsParams::default())
        .new_circuit_input_builder();
builder
    .handle_block(&geth_data.eth_block, &geth_data.geth_traces)
    .unwrap();
let block = block_convert(&builder.block, &builder.code_db);

let circuit = TestCircuit { block };
```

And then remove the `Default` from `Block` and from `TestCircuit`

Reply:
Thanks for the suggestions! Current version, once adding new field, compiler wont hint a implicit default setting, which rely on developer awareness. Removing default trait entirely eliminate this potiential mismatch from earth, and `block` should be created via empty block anyway.

Since many place rely on block `default` trait, such as super_circuit, exp_circuit, i will modify them accordingly :)

Reply:
Updated pull request. Only remove `Default` from `Block`, in order to keep consistent coding style for all circuit `without_witnesses` which still rely circuit to have `Default` trait. For `block` field in circuit, it will be wrapped with `Option<Block>`. 

Let me know whether you have better idea :)

Reply:
It does look a bit strange to have `Option<Block>` in the circuits, but then it makes sense that a circuit without the witness doesn't have a block.  I can't think of a better idea, so this way looks good to me :)